### ISSUE TITLE
regex issue with get_processor_name

### ISSUE THREAD

**Issue by TomW1605** [1337-server/automatic-ripping-machine#3]
### Description
the regex for linux in get_processor_name does not work on all machines. on my VM there is no "GHz" in the output of `cat /proc/cpuinfo`. this causes the regex to fail and an error:
```
File "/opt/arm/arm/ui/routes.py", line 501, in get_processor_name
speeds = str(speeds.group())
AttributeError: 'NoneType' object has no attribute 'group'
```

### Environment
DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=20.04
DISTRIB_CODENAME=focal
DISTRIB_DESCRIPTION="Ubuntu 20.04.1 LTS"

**Comment by 1337-server** [comment]
Would you mind sharing what the output of `cat /proc/cpuinfo`

I can fix the handeling of the error, but it would be nice to try fully fix this and see what youre getting that i missed.

**Comment by TomW1605** [comment]
this is for my VM
[cpuinfo.txt](https://github.com/1337-server/automatic-ripping-machine/files/5758798/cpuinfo.txt)
and this is for the host
[cpuinfo.txt](https://github.com/1337-server/automatic-ripping-machine/files/5758802/cpuinfo.txt)

**Comment by 1337-server** [comment]
Fixed with pull #7
